app/src/main/res/drawable/ic_clear_white_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```

Comment:
This file is not used in anywhere, so won't have any effect. 
Additionally ic_clear_black_24dp is used in answer_layout.xml at line 62, so we can't delete it. Even if it wasn't being in use, we still can't delete it in an irrelevant PR.